optaweb-tsp-planner/pom.xml

```diff
@@ -52,9 +52,43 @@
       <artifactId>spring-boot-starter-test</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.graphhopper</groupId>
+      <artifactId>graphhopper-reader-osm</artifactId>
+      <version>0.11.0</version>
+    </dependency>
   </dependencies>
 
   <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.pitest</groupId>
+          <artifactId>pitest-maven</artifactId>
+          <version>1.4.2</version>
+          <configuration>
+            <reportsDirectory>local/pit-reports</reportsDirectory>
+            <timestampedReports>true</timestampedReports>
+            <timeoutConstant>1000</timeoutConstant>
+            <mutators>
+              <!--
```

Comment:
May I ask why have been these mutators commented out?

Reply:
I agree that commented code is evil most of the times. This is for convenience when exploring various other mutators. So when you're satisfied with your mutation coverage and you wonder what other mutations might survive, you can just uncomment on of those mutators without a) searching for them in Pitest documentation, b) copy-pasting the code.

The reason why they're disabled is that these mutations are not that dangerous and important to cover and so they would just clutter the report with mutations that we don't really care about at this point.

Do you think adding an explanation like this is sufficient or it's better to remove the commented code?

Reply:
Let's keep it there if it's convenient for you and add some explanation. I was rather curious what's the reason - although I suspected it's probably there because of  experimenting, it's good to have it written there.

Reply:
My mistake. I checked out the docs and found out that the commented out mutators are part of the `DEFAULT` group so they're enabled now. I have removed them.